mne/viz/_3d.py

```diff
@@ -353,12 +353,56 @@ def plot_trans(info, trans_fname='auto', subject=None, subjects_dir=None,
     return fig
 
 
+def _percent_to_control_points(limits, stc_data, colormap):
+    """Private helper function to convert percentiles to control points.
+
+    Note: If using 'mne_analyze', generate cmap control points for a directly
+    mirrored cmap for simplicity. I.e., no normalization to account for a
+    2-tailed mne_analyze cmap is computed.
+
+    Parameters
+    ----------
+    limits : str | 3-tuple | dict
+        Desired percentages used to set cmap control points.
+
+    Returns
+    -------
+    ctrl_pts : array
+        Array of floats corresponding to values to use as cmap control points.
+    """
+
+    # Based on type of limits specified, get cmap control points
+    if limits == 'auto':
+        ctrl_pts = np.percentile(np.abs(stc_data), [97, 99.9])
+        ctrl_pts.insert(1, np.average(ctrl_pts))
+    elif isinstance(limits, tuple):
+        assert len(limits) == 3, '"limits" tuple must be length 3'
```

Comment:
this appears very non-standard to me. I would raise a `ValueError` or something of that sort. Maybe @Eric89GXL or @dengemann can confirm?